html/changelogs/Paprka-crades.yml.txt

```diff
@@ -0,0 +1,6 @@
+author: Paprka
```

Comment:
Honestly when I started it, it was mostly paprka's work now I'm not so sure due to all the lines I had to change to make the merge. I'm still thinking of making another change that will be moving my proc in crates to structures so I can also use it in the largecrate.